src/server/game/Entities/Player/Player.cpp

```diff
@@ -7534,6 +7534,16 @@ void Player::SendLoot(ObjectGuid guid, LootType loot_type)
     PermissionTypes permission = ALL_PERMISSION;
 
     LOG_DEBUG("loot", "Player::SendLoot");
+
+    const AuraType toRemove[] = {SPELL_AURA_MOD_STEALTH, SPELL_AURA_MOD_INVISIBILITY, SPELL_AURA_FEIGN_DEATH};
+    for (const auto& aura : toRemove)
+    {
+        if (HasAuraType(aura))
```

Comment:
Checking if they have the aura is not necessary, if they don’t when you try to remove it nothing happens

Reply:
I agree with Nyeriah